pkg/scheduler/framework/runtime/framework.go

```diff
@@ -402,13 +412,21 @@ func registerClusterEvents(name string, eventToPlugins map[framework.ClusterEven
 // a default from the scheme. Returns `nil, nil` if the plugin does not have a
 // defined arg types, such as in-tree plugins that don't require configuration
 // or out-of-tree plugins.
-func getPluginArgsOrDefault(pluginConfig map[string]runtime.Object, name string) (runtime.Object, error) {
+func getPluginArgsOrDefault(componentConfigVersion string, pluginConfig map[string]runtime.Object, name string) (runtime.Object, error) {
 	res, ok := pluginConfig[name]
 	if ok {
 		return res, nil
 	}
 	// Use defaults from latest config API version.
-	gvk := v1beta2.SchemeGroupVersion.WithKind(name + "Args")
+	var gvk schema.GroupVersionKind
+	switch componentConfigVersion {
+	case v1beta1.SchemeGroupVersion.String():
+		gvk = v1beta1.SchemeGroupVersion.WithKind(name + "Args")
+	case v1beta2.SchemeGroupVersion.String():
```

Comment:
we should do empty equivalent to v1beta2, otherwise we have to specify the version in every test.